src/lib/utils/sendBeacon.ts

```diff
@@ -45,9 +45,11 @@ const sendBeacon = (manifest: IManifest): boolean => {
 };
 
 const send = async (manifest: IManifest) => {
-    if (!(sendBeacon(manifest))) {
+    if (manifest.forceXHR === true) {
+        dbg(`Sending data directly with "XHR" because forceXHR is set to true.`);
+        await sendHttp(manifest);
+    } else if (!(sendBeacon(manifest))) {
```

Comment:
You are using the same method with the same input - why if - elseif ?

Reply:
Different message in the browser if debug mode is enabled...

Reply:
Deserves a comment